index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
- [ ] You did a great job @over-geek 😃, however,   I think you should sound more suggestive. so instead of `please write...` you can say `Kindly consider writing...` to be more suggestive and polite. 👍 
- [ ] You did a great job @maths29  😃, however, kindly make sure to add `why` the student should make a change in all comments you gave 👍 
- [ ] Also, it could be better to add emojis 😄 ,  to make your comment more friendly, Kindly add it 👍